string-constants-lib/string-constants/private/bulgarian-string-constants.rkt

```diff
@@ -1,6 +1,6 @@
-;; Bulgarian translation of Racket string constants file, version: 1.47
+;; Bulgarian translation of Racket string constants file, version: 1.52
 ;; This file is distributed under the same terms as Racket
-;; Copyright on translation: Alexander Shopov <ash@kambanaria.org>, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2024.
+;; Copyright on translation: Alexander Shopov <ash@kambanaria.org>, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2024р 2025.
```

Comment:
I think there is a typo here. The `p` should be a `,`.

Reply:
Fixed: https://github.com/racket/string-constants/pull/70